### PR TITLE
New package: grml-zsh-config-0.15.1

### PR DESCRIPTION
**Pull request by the-wrench99**

### CODE REVIEW

File: srcpkgs/grml-zsh-config/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'grml-zsh-config'
+pkgname=grml-zsh-config
+version=0.15.1
+revision=1
+noarch=1
+wrksrc=grml-etc-core-${version}
+short_desc="Grml's zsh setup"
```
Comment (maxice8) [inline srcpkgs/grml-zsh-config/template]: Why we packaging someone's dotfiles ?

File: srcpkgs/grml-zsh-config/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+maintainer="Christian Poulwey <christian.poulwey@t-online.de>"
+license="GPL-2.0-only"
+homepage="https://grml.org/zsh/"
+distfiles="https://deb.grml.org/pool/main/g/grml-etc-core/grml-etc-core_${version}.tar.gz"
+checksum=f6b79cdddc3a9ac1431c83f7dfef435de8dc61a56c1dc20e53e71fa376e4242e
+
+do_install() {
+	vmkdir		etc/skel
```
Comment (maxice8) [inline srcpkgs/grml-zsh-config/template]: no need for the vmkdir , vinstall can handle the directory creation.

File: srcpkgs/grml-zsh-config/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'grml-zsh-config'
+pkgname=grml-zsh-config
+version=0.15.1
+revision=1
+noarch=1
```
Comment (maxice8) [inline srcpkgs/grml-zsh-config/template]: it is more common to use noarch=yes

**Comment by maxice8** [comment]
Please notify when proposed changes are made.

File: srcpkgs/grml-zsh-config/template (hunk 3 above)
Comment (the-wrench99) [reply srcpkgs/grml-zsh-config/template]: done

**Pull request merged**